opencxl/cxl/component/root_complex/home_agent.py

```diff
@@ -102,7 +85,6 @@ def __init__(self, config: HomeAgentConfig):
 
         # emulated .mem s2m channels
         self._cxl_channel = {"s2m_ndr": Queue(), "s2m_drs": Queue(), "s2m_bisnp": Queue()}
```

Comment:
Although you didn't change this code, I would change this to either a dataclass or at least use Enum instead of using strings for the dictionary keys.

Reply:
Filed https://github.com/opencxl/opencxl/issues/102 to track.